packages/documentation/.storybook/addons/figma-link/FigmaLink.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useStorybookApi } from '@storybook/manager-api';
+
+import logoFigma from '../../../public/assets/images/technologies/logo-figma.svg';
+import { getTitleFromPath } from '../../../src/utils';
+
+function FigmaLink() {
+  const api = useStorybookApi();
+  const designParameter = api.getCurrentParameter<{ type: string; url: string }>('design') || [];
+  const titlePath = api.getCurrentStoryData()?.title;
+
+  return designParameter?.type === 'figma' && designParameter?.url ? (
+    <a className="figma-link btn btn-primary btn-sm btn-icon" href={designParameter.url}>
```

Comment:
```suggestion
    <a className="figma-link btn btn-secondary btn-sm btn-icon" href={designParameter.url}>
```
Perhaps, it's slightly better with secondary type.